goal_src/pc/engine/ui/progress/progress-draw.gc

```diff
@@ -1774,19 +1774,21 @@
                    ((= v1-81 1)
                     (draw-options-list obj font-ctx s2-1 *language-name-remap*))
                    ((= v1-81 #x10)
-                    (draw-options-list obj font-ctx s2-1 *pc-graphics-aspect-ratio-mode-remap*))
+                    (if (-> *pc-settings* use-original-aspect-ratio?)
+                      (draw-options-list obj font-ctx s2-1 *pc-graphics-original-aspect-ratio-mode-remap*)
+                      (draw-options-list obj font-ctx s2-1 *pc-graphics-aspect-ratio-mode-remap*)))
                    ((= v1-81 #x11)
                     (case (-> *pc-settings* aspect-ratio-mode)
-                          (('pc-aspect-4x3)
-                           (draw-options-list obj font-ctx s2-1 *pc-graphics-4x3-valid-resolutions-names*))
-                          (('pc-aspect-5x4)
-                           (draw-options-list obj font-ctx s2-1 *pc-graphics-5x4-valid-resolutions-names*))
-                          (('pc-aspect-16x9)
-                           (draw-options-list obj font-ctx s2-1 *pc-graphics-16x9-valid-resolutions-names*))
-                          (('pc-aspect-21x9)
-                           (draw-options-list obj font-ctx s2-1 *pc-graphics-21x9-valid-resolutions-names*))
-                          (('pc-aspect-32x9)
-                           (draw-options-list obj font-ctx s2-1 *pc-graphics-32x9-valid-resolutions-names*))))
+                      (('pc-aspect-4x3 'orig-aspect-4x3)
+                        (draw-options-list obj font-ctx s2-1 *pc-graphics-4x3-valid-resolutions-names*))
+                      (('pc-aspect-5x4)
+                        (draw-options-list obj font-ctx s2-1 *pc-graphics-5x4-valid-resolutions-names*))
+                      (('pc-aspect-16x9 'orig-aspect-16x9)
+                        (draw-options-list obj font-ctx s2-1 *pc-graphics-16x9-valid-resolutions-names*))
+                      (('pc-aspect-21x9)
+                        (draw-options-list obj font-ctx s2-1 *pc-graphics-21x9-valid-resolutions-names*))
+                      (('pc-aspect-32x9)
+                        (draw-options-list obj font-ctx s2-1 *pc-graphics-32x9-valid-resolutions-names*))))
```

Comment:
This menu stuff sure is messy, but if it works then it's good for now. We'll have to deal with it until we rewrite more of it.